src/test/java/com/connexta/search/index/IndexTests.java

```diff
@@ -18,11 +18,11 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.multipart;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+import com.connexta.search.index.controllers.IndexController;
```

Comment:
✏️ Same comment here -> Since this is the only ACCEPT_VERSION_HEADER_NAME is used from IndexController just static import

Reply:
See https://github.com/connexta/ion-search/pull/54#discussion_r335072853.